docs/operators-and-hooks-ref.rst

```diff
@@ -360,12 +360,19 @@ These integrations allow you to perform various operations within the Amazon Web
      -
      -
 
-   * - `Amazon EC2 <https://aws.amazon.com/ec2/>`__
+   * - `Amazon ECS <https://aws.amazon.com/ecs/>`__
      -
      -
      - :mod:`airflow.providers.amazon.aws.operators.ecs`
      -
 
+   * - `Amazon EC2 <https://aws.amazon.com/ec2/>`__
+     -
+     - :mod:`airflow.providers.amazon.aws.hooks.ec2`
+     - :mod:`airflow.providers.amazon.aws.operators.ec2_start_instance`,
+       :mod:`airflow.providers.amazon.aws.operators.ec2_stop_instance`,
+     - :mod:`airflow.providers.amazon.aws.sensors.ec2_instance_state`
+
```

Comment:
Can you put this block before `Amazon ECS`, please?

Reply:
Sure, I did not noticed that they are alphabetical